conf/messages.en

```diff
@@ -462,14 +462,11 @@ businessDetailsDoNotMatch.p2 = If entering your UTR again does not work, you can
 businessDetailsDoNotMatch.p2.link.text = find a lost UTR
 businessDetailsDoNotMatch.h2.2 = Try entering your business name again
 businessDetailsDoNotMatch.p3 = Check if you have entered your full business name as it was registered. You can find the registered business name on your incorporation certificate or governing document.
-businessDetailsDoNotMatch.p4 = Some businesses:
-businessDetailsDoNotMatch.li.1 = are known by a trading name that is different to their registered name
-businessDetailsDoNotMatch.li.2 = use abbreviations in their trading or registered name
-businessDetailsDoNotMatch.p5 = If your business is registered with Companies House, you can also
-businessDetailsDoNotMatch.p5.link.text = search the register for your company name
+businessDetailsDoNotMatch.p4 = If your business is registered with Companies House, you can also
+businessDetailsDoNotMatch.p4.link.text = search the register for your company name
 businessDetailsDoNotMatch.h2.3 = Contact HMRC to help us find your business
-businessDetailsDoNotMatch.p6.1 = If you have checked and entered your UTR and name again and we still cannot find your business, you can
-businessDetailsDoNotMatch.p6.2 = contact HMRC
+businessDetailsDoNotMatch.p5.1 = If you have checked and entered your UTR and name again and we still cannot find your business, you can
```

Comment:
Should be updated to:
businessDetailsDoNotMatch.p5.1 = If you have checked and entered your UTR and business name again and we still cannot find your business, you can

Reply:
Also, can we rename the keys to:
businessDetailsDoNotMatch.p5 = ...
businessDetailsDoNotMatch.p5.link.text = ...